doc/manual/metaprogramming.rst

```diff
@@ -9,106 +9,120 @@
 The strongest legacy of Lisp in the Julia language is its metaprogramming
 support. Like Lisp, Julia represents its own code as a data structure of
 the language itself.
+Because all data types and code in Julia are represented by Julia data structures,
+powerful reflection capabilities are available to explore the internals of a
+program and its types just like any other data.
 Since code is represented by objects that can be created and manipulated
```

Comment:
It seems to me that this sentence says almost exactly the same thing as the (new) one before it. Can they be re-worked into one, or maybe even just scratch one of them?